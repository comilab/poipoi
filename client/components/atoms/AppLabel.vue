<template lang="pug">
label.text-sm.font-bold.py-2
  slot
</template>

<script type="ts">
import { defineComponent } from 'nuxt-composition-api'

export default defineComponent({})
</script>
