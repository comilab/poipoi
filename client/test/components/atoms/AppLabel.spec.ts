import { render } from '@testing-library/vue'

import AppLabel from '~/components/atoms/AppLabel.vue'

describe('components/atoms/AppLabel', () => {
  test('renders correctly', () => {
    const { container, getByLabelText } = render(AppLabel, {
      slots: {
        default: `
          label
          <input />
        `
      }
    })

    expect(container).toMatchSnapshot()

    getByLabelText('label')
  })
})
