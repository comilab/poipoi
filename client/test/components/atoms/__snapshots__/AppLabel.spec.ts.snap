// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/AppLabel renders correctly 1`] = `
<div>
  <label
    class="text-sm font-bold py-2"
  >
    
          label
          
    <input />
  </label>
</div>
`;
